packages/cli/src/commands/apps/settings.ts

```diff
@@ -17,6 +17,13 @@ export default class AppSettingsCommand extends APICommand<typeof AppSettingsCom
 		description: 'the app id',
 	}]
 
+	static examples = [
+		{
+			description: 'get the settings of the app with the given id',
+			command: 'smartthings apps:settings 392bcb11-e251-44f3-b58b-17f93015f3aa',
+		},
+	]
```

Comment:
I'm not sure if we should do this on every command that supports it but this one (as well as most commands that show info but don't actually change something) allows using an index rather than an id. For example, you can specify `smartthings app:settings 3` and it will give details on the third app that is listed using `smartthings apps`. This command also will present a list and prompt if you leave the app out entirely.

Reply:
I think those are great cases to document in the examples.  It's otherwise not clear to the user which commands need you to specify an ID, or an input file, or whatever.  I like erring on the side of hand-holdy for help documentation.